src/field_theory/adjoin.lean

```diff
@@ -198,6 +198,14 @@ lemma _root_.alg_hom.map_field_range {K L : Type*} [field K] [field L] [algebra
   (f : E →ₐ[F] K) (g : K →ₐ[F] L) : f.field_range.map g = (g.comp f).field_range :=
 set_like.ext' (set.range_comp g f).symm
 
+lemma _root_.alg_hom.field_range_eq_top {K : Type*} [field K] [algebra F K] {f : E →ₐ[F] K} :
+  f.field_range = ⊤ ↔ function.surjective f :=
+set_like.ext'_iff.trans set.range_iff_surjective
+
+lemma _root_.alg_equiv.field_range_eq_top {K : Type*} [field K] [algebra F K] (f : E ≃ₐ[F] K) :
```

Comment:
For `linear_equiv` this is called `linear_equiv.range`, which IMO is a worse name. Would you mind renaming it to match your better name?

Reply:
Could I do this in a separate PR, since it might take a few rounds to find all of the occurrences?